views/routes/routes.js

```diff
@@ -43,6 +43,9 @@ export const apiRoutes = {
     users: {
       show: (uid) => `${apiPath}/admin/users/${uid}`,
     },
+    studies: {
+      all: `${apiPath}/admin/search/studies`,
```

Comment:
```suggestion
      index: `${apiPath}/admin/search/studies`,
```

Reply:
@Elkrival I'm with Mike on this one, currently we have `users.index` and I think it'll be annoying to guess between `index` and `all`